pkgs/development/python-modules/python-utils/default.nix

```
@@ -7,20 +7,22 @@
 , pytest-mypy
 , pytestCheckHook
 , pythonOlder
+, setuptools
+, typing-extensions
 }:
 
 buildPythonPackage rec {
   pname = "python-utils";
-  version = "3.5.2";
-  format = "setuptools";
+  version = "3.6.0";
+  format = "pyproject";
```

```suggestion
  format = "setuptools";
```
The upstream `pyproject.toml` does not define `[build-system]`, so the format is still `setuptools`.

---

Good catch! I reverted it to setuptools. Thanks!